packages/components/src/local/organisms/planning-force/index.tsx

```diff
@@ -1,14 +1,18 @@
 import cx from 'classnames'
-import L, { LatLng, latLng } from 'leaflet'
-import React, { useCallback } from 'react'
+import L, { LatLng, latLng, LeafletMouseEvent } from 'leaflet'
+import React, { useEffect } from 'react'
 import * as ReactDOMServer from 'react-dom/server'
-import { LayerGroup, Marker, Tooltip } from 'react-leaflet-v4'
+import { LayerGroup, useMap } from 'react-leaflet-v4'
 import AssetIcon from '../../asset-icon'
 import { AssetRow } from '../planning-assets/types/props'
 import styles from './styles.module.scss'
+import 'leaflet.markercluster/dist/leaflet.markercluster'
+import 'leaflet.markercluster/dist/MarkerCluster.css'
+import 'leaflet.markercluster/dist/MarkerCluster.Default.css'
 import PropTypes from './types/props'
 
-export const PlanningForces: React.FC<PropTypes> = ({ assets, selectedAssets, setSelectedAssets, interactive }) => {
+const PlanningForces: React.FC<PropTypes> = ({ assets, selectedAssets, setSelectedAssets, interactive }) => {
+  const mcg = L.markerClusterGroup()
```

Comment:
@lilitkarapetyan - does this create `mcg` on each render?  I'm surprised we aren't storing it in some `state-aware` structure so we know it is only created once.